script.js

```diff
@@ -0,0 +1,90 @@
+    }
+
+    else {
+    document.getElementById('item_nav').style.display = 'flex'
+    document.getElementById('header').style.display = 'flex'
+    document.getElementsByClassName('hamburger_button')[0].background = "none"
+    document.getElementById('hamburger_button_x').style.display = "block"
+    document.getElementsByTagName('body')[0].style.overflow = 'hidden'
+    is_on = true
+    }
+})
+
+function close_button() {
+    if (screen.width < 768) {
+    document.getElementById('item_nav').style.display = 'none'
+    document.getElementById('header').style.display = 'none'
+    document.getElementById('hamburger_button_x').style.display = "none"
+    document.getElementsByTagName('body')[0].style.overflow = 'scroll'
+    is_on = false
+    }
+}
+
+
+const card_info = {
+    image: ["./assets/1Placeholder.svg", "./assets/2ndPlaceholder.svg", "./assets/3rdPlaceholder.svg", "./assets/1Placeholder.svg", "./assets/2ndPlaceholder.svg", "./assets/3rdPlaceholder.svg"],
+    name: ["Profesional Art Printing Data More", "Data Dashboard Healthcare", "Website Protfolio", "Profesional Art Printing Data More", "Data Dashboard Healthcare", "Website Protfolio"],
+    description: ["A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard","A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard","A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard","A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard","A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard","A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard"],
+    live_link: ["https://rawcdn.githack.com/glitcher255/micro-porfolio/3aef2ff8602559c098666dcf4522e7baa82ca27c/index.html","https://rawcdn.githack.com/glitcher255/micro-porfolio/3aef2ff8602559c098666dcf4522e7baa82ca27c/index.html","https://rawcdn.githack.com/glitcher255/micro-porfolio/3aef2ff8602559c098666dcf4522e7baa82ca27c/index.html","https://rawcdn.githack.com/glitcher255/micro-porfolio/3aef2ff8602559c098666dcf4522e7baa82ca27c/index.html","https://rawcdn.githack.com/glitcher255/micro-porfolio/3aef2ff8602559c098666dcf4522e7baa82ca27c/index.html","https://rawcdn.githack.com/glitcher255/micro-porfolio/3aef2ff8602559c098666dcf4522e7baa82ca27c/index.html"],
+    source_link: ["https://github.com/DavidCazco36/My-Portfolio", "https://github.com/DavidCazco36/My-Portfolio", "https://github.com/DavidCazco36/My-Portfolio", "https://github.com/DavidCazco36/My-Portfolio", "https://github.com/DavidCazco36/My-Portfolio", "https://github.com/DavidCazco36/My-Portfolio"],
+    technologies: [{
+        tech_stack:["html", "bootstrap", "Ruby"]
+    }]
+    }
+    
+
+
```

Comment:
- [ ] The projects should be an array of objects, check the code below, this is considered one project, please add them this way 
![image](https://user-images.githubusercontent.com/62705947/208995013-a6649bcd-a6b3-4ba4-9114-2fe5d2270268.png)

Reply:
It already is an array of objects, I simply added the other ones in as well, is this a joke?

Reply:
- [ ] Hey @glitcher255, I missed the array inside the objects, I sincerely apologize for this honest mistake, you are good to go for this requirement :)